docker-compose.yml

```diff
@@ -1,5 +1,5 @@
 services:
-    learnhub.test:
+    laravel.test:
```

Comment:
What's the idea behind this change?

Reply:
by defaul sail don't work if the name is change, in case i m missing something... please advice

Reply:
It didn't work because the `APP_PORT` and `APP_SERVICE` were missing from the .env.example and .env files but I added them yesterday so it should be good now.